packages/code-snippet/src/CodeSnippetWidget.tsx

```diff
@@ -79,7 +79,7 @@ class CodeSnippetTable extends React.Component<ICodeSnippetProps> {
           tooltip={codeSnippet.description}
           actionButtons={actionButtons}
         >
-          <pre>{codeSnippet.code.join('\n')}</pre>
+          <textarea>{codeSnippet.code.join('\n')}</textarea>
```

Comment:
I see the following error in my console:

```
Warning: Use the `defaultValue` or `value` props instead of setting children on <textarea>.
    in textarea (created by CodeSnippetTable)
    in div (created by ExpandableComponent)
    in div (created by ExpandableComponent)
    in ExpandableComponent (created by CodeSnippetTable)
    in div (created by CodeSnippetTable)
    in div (created by CodeSnippetTable)
    in div (created by CodeSnippetTable)
    in div (created by CodeSnippetTable)
    in CodeSnippetTable (created by UseSignal)
    in UseSignal
    in div
```

Reply:
updated https://github.com/elyra-ai/elyra/pull/431/commits/78b43e0211f7e96d2bb2d31af07acbde39e405ac